test_ncbi.py

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env python3
+
+import taxonomy
+from downloads import download
+import subprocess
+import unittest
+
+
+class LatestNCBITestCase(unittest.TestCase):
+    def test_load_latest_ncbi_taxonomy(self):
```

Comment:
It downloads data from external uri and creates a file in user's file system.

Maybe it should have a `@unittest.skipIf` and env flag?

Reply:
Agreed, unit tests contacting external services should probably not run by default

Reply:
done!